pslr_scsi.c

```diff
@@ -23,7 +23,9 @@
     and GNU Lesser General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
+#ifdef RAD10
+#include "pslr_scsi_win.c"
+#else
```

Comment:
We need this because WIN32 is not defined if we compile RAD10? Interesting. Can you merge the two ifdefs?

Reply:
Actually RAD10 defines _WIN32, not WIN32.

About merging the ifdefs. Do you mean something like: #if defined(RAD10) || defined(WIN32)
I wasn't sure that would compile on all the platforms.

Reply:
Changed to #if defined(WIN32) || defined(RAD10)

Reply:
_WIN32 instead of WIN32? That's nice. Yes, this is the change I was asking for.